api/nakadi-event-bus-api.yaml

```diff
@@ -995,11 +1014,42 @@ definitions:
         type: string
         default: 'random'
 
+      compatibility_mode:
```

Comment:
Nakadi will check it and ensure that no major changes are accepted if the compatibility mode is forwards.